.docker/nginx/sites/opendor.me.conf

```diff
@@ -0,0 +1,29 @@
+upstream php-upstream {
+   server app:9000;
+}
+
+server {
+    listen 80;
+
+    index index.php index.html;
+
+    error_log  /var/log/nginx/error.log;
+    access_log /var/log/nginx/access.log;
```

Comment:
Could we write to `/var/www/storage/logs/nginx/...` here? 🤔
That would allow everyone to see the logs locally without entering the container. And the whole folder content is git ignored so no additional config in the project.

Reply:
Looks like nice idea, Updating